classes/03class/exercises/c03-docker03/gpavelar/README.md

```diff
@@ -0,0 +1,46 @@
+# C03-Docker03
+
+## Commands
+
+- Commands executed to push both images to DockerHub as well as the output from those commands:
+
+```bash
+## I've already done another example how to push image to dockerhub, so
+## all my credentials were set up.
+## Docker images
+docker images
+REPOSITORY            TAG                 IMAGE ID            CREATED             SIZE
+exercise02            v2                  fcf95bce2dd9        21 hours ago        107MB
+exercise02            v1                  73a3b15c384b        23 hours ago        107MB
+
+## Tagging image
+docker tag 73a3b15c384b gpavelar/exercise02:v1
```

Comment:
You don't need to specify the image id hash, you could use the current tag as source `exercise02:v1`.

Reply:
Thanks for your time and review comments  :+1: